src/components/common/Buttons/FormButton.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const FormButton = props => {
+  const { value, onClick, disabled } = props;
+  return (
+    <input
```

Comment:
Why are you using a different input component if the team has created some reusable inputs to use?

Reply:
Again I think the inputs shouldn't be in buttons directory because we have the `Inputs` directory

Reply:
These are three different components we have, and I think they are not the same. Also, there is no way I can use TextInput as button since it doesn't not have same properties as FormButton. 

<img width="735" alt="Screen Shot 2019-04-15 at 13 17 30" src="https://user-images.githubusercontent.com/20681465/56128847-0fae1e80-5f81-11e9-8c78-fbe1ef9311ff.png">
<img width="735" alt="Screen Shot 2019-04-15 at 13 17 38" src="https://user-images.githubusercontent.com/20681465/56128848-0fae1e80-5f81-11e9-869e-120e8a7f6713.png">
<img width="472" alt="Screen Shot 2019-04-15 at 13 18 57" src="https://user-images.githubusercontent.com/20681465/56128849-0fae1e80-5f81-11e9-8b76-2b9e66f7fc13.png">